src/main/java/dev/latvian/mods/kubejs/client/KeybindRegistryKubeEvent.java

```diff
@@ -0,0 +1,74 @@
+package dev.latvian.mods.kubejs.client;
+
+import com.mojang.blaze3d.platform.InputConstants;
+import dev.latvian.mods.kubejs.util.ClassWrapper;
+import dev.latvian.mods.rhino.util.HideFromJS;
+import net.minecraft.client.KeyMapping;
+import net.neoforged.neoforge.client.settings.KeyConflictContext;
+import net.neoforged.neoforge.client.settings.KeyModifier;
+import org.lwjgl.glfw.GLFW;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class KeybindRegistryKubeEvent implements ClientKubeEvent {
+	private final List<Builder> builders = new ArrayList<>();
+
+	public ClassWrapper<GLFW> getGLFW() {
```

Comment:
This method name results in a bean of `gLFW`, not really ideal. Either the method name should be `getGlfw` or the class wrapper is a global client binding of `GLFW` (so you could do `GLFW.KEY_A`).

Reply:
I think a public final field of ClassWrapper<GLFW> should also work?